firefox-ios/l10n-screenshots.sh

```diff
@@ -46,7 +46,7 @@ for lang in $LOCALES; do
     if [ "$?" != "0" ]; then
         echo "Fastlane exited with code: $?"
         exit $?
-    elif grep -q "** TEST FAILED **"; then
+    elif grep -q "TEST FAILED" "output.txt"; then
```

Comment:
"output.txt" was missing previously, so this line might not be working properly before.